### DISCUSSION TITLE
What's the correct way to edit image source, which is also a link?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#856] [Q&A] [answered]
Hi 

What's the correct way to edit image source, which is also a link?

I created this [codepen](https://codepen.io/galinhalx/pen/rNJGByz), but I'm not able to popup the image editor (it appears and disappears quickly)

Is this related to the bug of editing `<a> `despite I added temporary fix from Lea, or am I missing something on the way to do it?

**Comment by DmitrySharabin** [comment, marked as answer]
In what browsers in what OS are you facing this issue? I can't reproduce it. It looks like it's working. I checked in Safari and Chrome on macOS.

<img width="598" alt="Screen Shot 2022-05-23 at 22 52 20" src="https://user-images.githubusercontent.com/9166277/169905202-affbb240-28ee-498e-b665-4e5583e209e1.png">


However, I can see a bunch of CORS errors in the console.

**Safari**
<img width="1611" alt="Screen Shot 2022-05-23 at 22 51 52" src="https://user-images.githubusercontent.com/9166277/169905630-39cc8d08-87d5-4b92-948a-d8245f6bbef4.png">

**Chrome**
<img width="1474" alt="Screen Shot 2022-05-23 at 22 56 17" src="https://user-images.githubusercontent.com/9166277/169905649-0d2eabd4-9756-4205-8a82-76d1cb3744ff.png">


What do you see in yours? Is it possible that the issue is caused by the server setting?

**Reply by GalinhaLX** [reply]
Hi @DmitrySharabin 

mac OS Catalina 10.15.7
with Firefox 100.0.1 (64 bits), same with firefox 100.0.2 (64 bits) => Doesn't work

and you're right, works with 
Chrome (Version 101.0.4951.64 (Build officiel) (x86_64))
or
Safari Version 15.4 (15613.1.17.1.13, 15613)

Also updated image source to avoid CORS errors

**Reply by DmitrySharabin** [reply]
I can definitely confirm that the popup behaves weird in Firefox when an image property is inside an `<a>` element. Unfortunately, I have no idea what caused the issue. 🤷🏻‍♂️ However, I also noticed that if you hold the left mouse button before releasing it, the popup appears as it should.

**Reply by GalinhaLX** [reply]
Weird behaviour 
I guess I have to find a workaround if user is using firefox

By the way, thanks, I now know, that I'm doing it right with image links :)

**Reply by DmitrySharabin** [reply]
My pleasure! 🤝 Happy to help. 😃
